modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -43,22 +47,22 @@ This process requires that you have write access to a container image registry o
 endif::restricted[]
 
 .Prerequisites
-ifndef::openshift-rosa,openshift-dedicated[]
+ifndef::openshift-rosa,openshift-dedicated,oc-mirror-v2[]
 * You configured a mirror registry to use in your disconnected environment.
-endif::openshift-rosa,openshift-dedicated[]
+endif::openshift-rosa,openshift-dedicated,oc-mirror-v2[]
 ifdef::openshift-rosa,openshift-dedicated[]
 * You configured a mirror registry to use.
 endif::openshift-rosa,openshift-dedicated[]
-ifdef::restricted[]
+ifdef::restricted,oc-mirror-v2[]
```

Comment:
Also, I think you'd want to set :restricted: in the ifeval on line 23. oc-mirror v2 is also restricted, right? So if you add it there, then it will properly show this line without needing to add oc-mirror-v2 in this ifdef

Reply:
Have changed it.